### ISSUE TITLE
After uninstalling Atom, option "open with Atom" is shown

### ISSUE THREAD

**Issue by miparnisari** [atom/atom#12846]
### Description

I uninstalled Atom. I noticed that when I right clicked an expansion drive I got the "open with Atom" option, and then an error saying "Application not found".

![image](https://cloud.githubusercontent.com/assets/5374887/19017841/0d16d85e-8821-11e6-9b8b-1cbd96b56c31.png)

![image](https://cloud.githubusercontent.com/assets/5374887/19017843/0fceca0c-8821-11e6-96aa-ab6334548125.png)

**Comment by damieng** [comment]
This will happen if you upgrade to 1.10 and then uninstall.  The fix is in 1.11 but that's currently in beta.

If you want to manually clean up you can run these commands in the command prompt;

```
reg delete "HKEY_CURRENT_USER\Software\Classes\Applications\Atom" /f
reg delete "HKEY_CURRENT_USER\Software\Classes\*\shell\Atom" /f
reg delete "HKEY_CURRENT_USER\Software\Classes\Directory\shell\Atom" /f
reg delete "HKEY_CURRENT_USER\Software\Classes\Directory\background\shell\Atom" /f
```

**Comment by miparnisari** [comment]
Thanks @damieng! The last command did the trick.
